cogs/tickets.py

```diff
@@ -126,25 +134,25 @@ def topic_check(m):
                 icon_url=av,
                 text="IsThicc Tickets"
             )
-            return await dm_msg.edit(embed=timeout_e)
+            return await reaction_user.send(embed=timeout_e)
 
-        time = datetime.now().strftime("%A, %d %b %Y %l:%M %p")
-        category = discord.utils.get(guild.categories, name="『 Tickets 』")
+        category = discord.utils.get(guild.categories, id=0) # TODO: Change this to the ID of the tickets category
 
         ticket = await guild.create_text_channel(
-            name=f'Ticket-{reaction_user.display_name}',
+            name=f'ticket-{reaction_user.display_name}',
             category=category,
-            topic=f"Ticket opened by {reaction_user.mention} at {time}! Priority: {priority[1]}"
+            topic=f"Ticket opened by {reaction_user.mention}. Priority: {priority.value}"
```

Comment:
What is `priority.value`?

Reply:
Gets the value of the enum, so instead of outputting "Priority.LOW" it would output the value in this case "Low"

Reply:
Oh, yea I'm dumb, I didn't see when you assigned priority. In this case, you don't need `priority = None` above the `try:` by the way.